pumpkin-config/src/lib.rs

```diff
@@ -159,7 +160,10 @@ trait LoadConfiguration {
 
 impl LoadConfiguration for AdvancedConfiguration {
     fn get_path() -> &'static Path {
-        Path::new("features.toml")
+        match env::var("FEATURES") {
+            Ok(v) => Path::new(v.leak()),
```

Comment:
I'm assuming usage of `.leak()` is fine since `.get_path()` is called only once during the lifetime of the application.

It's also why I left the `match` condition alone, if the `match` condition were to exist on a hot code path I'd rather have seen a different solution.